### PR TITLE
:sparkles: Coverage Report

### PR DESCRIPTION
**Pull request by shnizzedy**
- :books: Generate coverage report
- :books: Send coverage report to [CodeCov](https://codecov.io/gh/ChildMindInstitute/mindlogger-app-backend)
- :green_heart: Allow that first user created is admin before creating first test user
- :pencil: fix a typo

### CODE REVIEW

File: test/testLib.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ def testCreateUser(girder, email=None):
     assert createUser['firstName'] == firstName, 'firstName does not match'
     assert createUser['lastName'] == lastName, 'lastName does not match'
     assert createUser['login'] == login, 'login does not match, {} {}'.format(createUser['login'], login)
-    assert createUser['admin'] == False, 'user is an admin'
+    if not admin:
+        assert createUser['admin'] == False, 'user is an admin'
```
Comment (henryrossiter) [inline test/testLib.py]: If ```admin == True```, should we test to make sure user is in fact an admin?

If that is necessary we could get rid of the conditional and just use: 
```assert createUser['admin'] == admin```

Comment (shnizzedy) [reply test/testLib.py]: @henryrossiter good idea! e9d9bf7

**Comment by shnizzedy** [comment]
Resolves https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/194

**Comment by shnizzedy** [comment]
https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/200/commits/3915a97db270d4a8c4f2d9ee19428c8c4e73d1d0 is running on both [API](https://api.mindlogger.org) & [dev](https://dev.mindlogger.org)

**Pull request merged**